tests/devices/qubit/test_measure.py

```diff
@@ -419,6 +419,8 @@ def test_jax_backprop(self, convert_to_hamiltonian, use_jit):
         """Test that backpropagation derivatives work with jax with hamiltonians and large sums."""
         import jax
 
+        jax.config.update("jax_enable_x64", True)
```

Comment:
do we still need this? thought we made it default. same for the rest of tests

Reply:
I have no idea why, but I was getting failures here until I added this.  I still have no idea why.